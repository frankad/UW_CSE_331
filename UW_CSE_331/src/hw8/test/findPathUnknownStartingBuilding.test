# Verify that when unknown starting building
# is entered by the user to find the path.So, that it will print "Unknown 
# building" followed by the building abbreviated name of the starting
# building. It will print "Unknown building: AA". AA is Abbreviated name
# of starting building.
r
ABX
PAB

# terminate the program
q