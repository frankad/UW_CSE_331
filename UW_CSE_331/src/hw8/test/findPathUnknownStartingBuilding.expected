Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify that when unknown starting building 
# is entered by the user to find the path.So, that it will print "Unknown 
# building" followed by the building abbreviated name of the starting
# building. It will print "Unknown building: AA". AA is Abbreviated name
# of starting building.
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: ABX

Enter an option ('m' to see the menu): 
# terminate the program